packages/expo-image/src/Image.types.ts

```diff
@@ -178,6 +178,18 @@ export interface ImageProps extends Omit<ViewProps, 'style' | 'children'> {
    */
   priority?: 'low' | 'normal' | 'high' | null;
 
+  /**
+   * Sets the HTML `loading` attribute on the `<img>` element.
```

Comment:
Might be worth linking to MDN docs
```suggestion
   * Sets the HTML [`loading`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/img#loading) attribute on the `<img>` element.
```